platform/openide.util/test/unit/src/org/openide/util/BaseUtilitiesTest.java

```diff
@@ -557,4 +567,141 @@ public void test_toPrimitiveArray_returnsArrayOfPrimitives_whenGivenArrayOfObjec
+    }
+    //--------------------------------------------------------------------------
+    public void test_getShortClassName_returnsClassName_forProperInvocation() {
+        
+        assertEquals("String", getShortClassName(String.class));
+        assertEquals("String[]", getShortClassName(String[].class));
+        assertEquals("UnicodeBlock", getShortClassName(Character.UnicodeBlock.class));
+    }
+    //--------------------------------------------------------------------------
+    public void test_pureCalssName_thworsNullPointer_whenGivenNullArgument() {
+        
+        try {
+            pureClassName(null);
+            fail();
+        } catch (final NullPointerException e) {
+            //good
+        }
+    }
+    //--------------------------------------------------------------------------
+    public void test_pureCalssName__returnsEmptyString_whenGivenEmptyString() {
+        
+        assertEquals("", pureClassName(""));
+    }
+    //--------------------------------------------------------------------------
+    public void test_pureCalssName__returnsBlankString_whenGivenBlankString() {
+        
+        assertEquals(" \t", pureClassName(" \t"));
+    }
+    //--------------------------------------------------------------------------
+    public void test_pureCalssName__returnsClassName_whenGivenFullyQualifiedClassName() {
+        
+        assertEquals("java.lang.String", pureClassName("java.lang.String"));
+        assertEquals("UnicodeBlock", pureClassName("java.lang.Character$UnicodeBlock"));
+    }
+    //--------------------------------------------------------------------------
+    // It seems wrapStringToArray is not used at all, maybe we could remove it all together
+    // https://commons.apache.org/proper/commons-text/ provides more or less the same functionality
+    //--------------------------------------------------------------------------
+    public void test_wrapStringToArray_throwsNullPointer_whenGivenNullArgument() {
+        
+        try {
+            wrapStringToArray(null, 11, BreakIterator.getLineInstance(), true);
+            fail();
+        } catch (final NullPointerException e) {
+            //good
+        }
+        try {
+            //this is slopy behavior - null pointer exception happens only
+            //when line breaking actually happenes - change second argument below 
+            // from 1 to 10 and and rerun the test to observe this
+            wrapStringToArray("ab\ncd", 1, null, true);
+            fail();
+        } catch (final NullPointerException e) {
+            //good
+        }
+    }
+    //--------------------------------------------------------------------------
+    public void test_wrapStringToArray_assumesWidthEqalsOne_whenGivenZeroWidth() {
+
+        String[] wrapped = wrapStringToArray("a\nb", 0, BreakIterator.getLineInstance(), true);
+        assertEquals(3, wrapped.length);
+        assertEquals("a ", wrapped[0]); // Why is there a space at the end?
```

Comment:
Quoting from [BreakIterator API](https://docs.oracle.com/javase/7/docs/api/java/text/BreakIterator.html#line): 

> The BreakIterator instances returned by the factory methods of this class are intended for use with natural languages only, not for programming language text.

Question is, are you sure `BreakIterator.getLineInstance()` (which is locale dependent, so would depend on the Locale the user is using to build NetBeans) is what you need here?

Reply:
Thanks for spotting this - I fixed it with new commit.